src/main/java/com/redhat/cloud/custompolicies/app/model/FullTrigger.java

```diff
@@ -51,6 +56,26 @@ public FullTrigger(Policy policy) {
     Condition cond = new Condition();
     cond.expression = policy.conditions;
     conditions.add(cond);
+    storeActions(trigger, policy);
+  }
+
+  private void storeActions(Trigger trigger, Policy policy) {
+    String[] actionsIn = policy.actions.split(";");
+    for (String actionIn : actionsIn) {
+      TriggerAction ta = new TriggerAction();
+      if (actionIn.startsWith("EMAIL")) {
```

Comment:
We actually use lower case for the action type:
https://github.com/RedHatInsights/custom-policies-ui-frontend/blob/cb562622c4cceb47b0ff2b898f112f6ac801e80c/src/types/Policy/Actions/ActionType.ts#L2-L3